schemas/org.openweathermap.snowplow/weather_stamp/jsonschema/1-0-0

```diff
@@ -0,0 +1,151 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
+    "vendor" : "org.openweathermap.snowplow",
+    "name" : "weather_stamp",
```

Comment:
I've explained it in scala-weather README. OpenWeatherMap uses this in forecast and history lookups. When we're making history/forecast request to OWM it returns JSON with technical info, city id, and list of these weather stamps. Basically it means "weather conditions for exact point in time" (dt represent timestamp). I searched correct name for it in OWM specs, but haven't found, so "weather stamp" is my own name for it.